LayoutTests/media/media-controller-play-then-pause.html

```diff
@@ -39,10 +39,12 @@
 	run('controller.play()');
 }
 
-function playAgain() {
+async function playAgain() {
 	consoleWrite("EVENT(play)");
 	testExpected('controller.playbackState', 'playing');
 	controller.removeEventListener('play', playAgain, true);
+  await sleepFor(2000);
```

Comment:
2 seconds is a lot... The test runner might timeout more easily now. Was there a specific reason for waiting that much?

Reply:
I wrote a new testing function for that. It works much better.

Reply:
Seems to fail on iOS :(

Reply:
Attempted fix.